### PR TITLE
refactor: remove `MCI` prefixes

### LINKED ISSUE #34: Remove ‘ManFisher’ and ‘Media Competence Institute’ references
The project is no longer under the Media Competence Institute org. (which appears to have been [deleted](https://github.com/media-competence-institute)), so Class names (and a couple of functions) should no longer be prefaced with `MCI_`.

Additionally, there is no mention anywhere in the codebase or Plugin directory about the former ‘ManFisher Media ManuFactory’, and the ManFisher dashboard note currently reads ‘the ManFisher menu Will soon disappear as the company name changed and I believe it's overbearing to have that menu for such a simple function as footnotes’.

The dashboard menu item should be moved under the ‘Settings’ menu and reduced to a single **footnotes** tab rather than the three current ones.

@markcheret to review and approve

### PR DESCRIPTION
**Pull request by Rumperuu**
Remove all `MCI` prefixes.

Close #34

### CODE REVIEW

**Comment by Rumperuu** [comment]
TODO: revert 1284544, and add type declaration and hinting _first_, which will mitigate against the risk of any further variable naming conflicts (see that commit message for more information).

File: class/convert.php [hunk 1]
```diff
@@ -50,62 +50,62 @@ public static function index( $p_int_index, $p_str_convert_style = 'arabic_plain
...
 	 * @return string
 	 *
 	 * Edited:
 	 * @since 2.2.0   optionally lowercase (code from Latin)
 	 */
-	private static function to_romanic( $p_int_value, $p_bool_upper_case ) {
+	private static function to_romanic( $value, $upper_case ) {
 		// Table containing all necessary romanic letters.
-		$l_arr_romanic_letters = array(
+		$romanic_letters = array(
```
Comment (markcheret) [inline class/convert.php:108]: Sorry to be nitpicky here.
I just checked this actually for the first time in the code.
Can we please change it to “roman numerals”?
I was confused and would like it to be clearer 😜

Comment (Rumperuu) [reply class/convert.php:108]: https://www.meme-arsenal.com/memes/ab44c2547188fec097731cc6005b4cd3.jpg

;)

**Comment by Rumperuu** [comment]
I'm limiting this PR to just the `MCI` prefixes, as this is the only outstanding task for the [2.7.1 release](https://github.com/markcheret/footnotes/milestone/3).

**Review by markcheret — APPROVED** [review]
Looking good. Thanks for your work.

**Pull request merged**